libs/sdk-ui-kit/src/List/InvertableSelect/test/useInvertableSelect.test.ts

```diff
@@ -153,25 +207,24 @@ describe("useInvertableSelect", () => {
                 5,
                 "abc",
             ],
-
             [
                 "none",
-                "not inverted and selection is empty and empty search and all items loaded",
+                "is not inverted and selection is empty and empty search and all items loaded",
                 [1, 2, 3, 4, 5],
                 [],
                 false,
                 5,
                 "abc",
             ],
-            [
+            /* [
                 "none",
                 "not inverted and selection is empty and empty search and all items not loaded",
```

Comment:
This commented out case should be removed?

Reply:
fixed